src/Hl7.Fhir.Base/Rest/FhirClientSettings.cs

```diff
@@ -128,6 +129,17 @@ public bool CompressRequestBody
         /// </summary>
         public ParserSettings? ParserSettings = ParserSettings.CreateDefault();
 
+        /// <summary>
+        /// How to transfer binary data when sending data to a Binary endpoint.
+        /// </summary>
+        public BinaryTransferBehaviour BinarySendBehaviour = BinaryTransferBehaviour.UseResource;
+
+        /// <summary>
+        /// How to transfer binary data when receiving data from a Binary endpoint.
```

Comment:
Maybe change the comment to: "Whether we ask the server to return us binary data or a Binary resource."

Reply:
Maybe it should be called "BinaryReceivePreference"